src/main/java/com/amazon/opendistroforelasticsearch/security/securityconf/Migration.java

```diff
@@ -96,7 +96,20 @@
         }
         return c7;
     }
-    
+
+    public static SecurityDynamicConfiguration<NodesDnV7> migrateNodesDn(SecurityDynamicConfiguration<NodesDnV6> n6) {
```

Comment:
I did merge the classes. However this migration section remains since the version is part of the configuration in v7 as Meta fields.

**V7**

```yml
_meta:
  type: "nodesdn"
  config_version: 2
cluster1:
  nodes_dn:
    - "CN=abc.com"
```

**V6**
```yml
cluster1:
  nodes_dn:
    - "CN=abc.com"
```